test/parallel/test-internal-errors.js

```diff
@@ -5,153 +5,64 @@ const common = require('../common');
 const assert = require('assert');
 const errors = require('internal/errors');
 
-function invalidKey(key) {
-  return new RegExp(`^An invalid error message key was used: ${key}\\.$`);
-}
-
-errors.E('TEST_ERROR_1', 'Error for testing purposes: %s');
-errors.E('TEST_ERROR_2', (a, b) => `${a} ${b}`);
+errors.E('TEST_ERROR_1', 'Error for testing purposes: %s',
+         Error, TypeError, RangeError);
+errors.E('TEST_ERROR_2', (a, b) => `${a} ${b}`, Error);
 
 {
-  const err = new errors.Error('TEST_ERROR_1', 'test');
+  const err = new errors.codes.TEST_ERROR_1('test');
   assert(err instanceof Error);
```

Comment:
When being on it: this is actually not a good test because e.g., `TypeError` is going to be instanceof Error as well. I would change that to test for the name.

Reply:
The error name is checked on the line below this one